ModMenu/Settings/HeaderFix.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Kingmaker.PubSubSystem;
+using Kingmaker;
+using UnityEngine;
+using Kingmaker.Utility;
+using HarmonyLib;
+using Kingmaker.Blueprints.JsonSystem;
+
+namespace ModMenu.Settings
+{
+  internal class HeaderFix
+  {
+    [HarmonyPatch(typeof(BlueprintsCache))]
+    static class BlueprintsCache_Patch
+    {
+      [HarmonyPatch(nameof(BlueprintsCache.Init)), HarmonyPostfix]
+      static void Postfix()
+      {
+        EventBus.Subscribe(new Fix());
+      }
+    }
+  }
+
+  internal class Fix : ISettingsUIHandler
+  {
+    public void HandleOpenSettings(bool isMainMenu = false)
+    {
+      foreach (RectTransform transform in Game.Instance.RootUiContext.m_CommonView?.transform.Find("Canvas/SettingsView/ContentWrapper/MenuSelectorPCView"))
```

Comment:
🤦‍♂️

Reply:
I mean I love the fix but also... y owlcat. y.

Reply:
I couldn't for the life of me see why the scale would be changed in game, but not in the main menu...